example/src/test/resources/TestTool/ManualExerciseMundo/scenario01.properties

```
@@ -0,0 +1,15 @@
+scenario.description = Test input validation
+
+adapter.validateDocument.className = nl.nn.adapterframework.senders.IbisJavaSender
+adapter.validateDocument.serviceName = testtool-ManualExerciseMundo
+
+step1.adapter.validateDocument.write = valid.xml
+step2.adapter.validateDocument.read = ok.txt
```

```suggestion
step2.adapter.validateDocument.readline = ok
```
Je kan in plaats van bestanden ook regels direct lezen.